src/components/App.js

```diff
@@ -85,7 +85,7 @@ class App extends Component {
                                 <Navbar.Brand href="/">
                                             <img
                                                 src={logo}
-                                                width="50"
```

Comment:
these proportions (50 to 100) are not the same as the proportions of the image (1380px to 2549px), so it appears a bit blurry. Could you either crop the image to 1380px by 2760px by adding extra padding on left and right or use the same proportions as the image?